src/etc/htmldocck.py

```diff
@@ -365,6 +365,7 @@ def check_tree_attr(tree, path, attr, pat, regexp):
 
 def check_tree_text(tree, path, pat, regexp):
     path = normalize_xpath(path)
+    print("normalize_xpath -> {}".format(path))
```

Comment:
Perhaps this was a debugging change left in by accident? You didn't mention it in the PR description.

Reply:
ah, yes that is a debugging artefact '^^